CHANGELOG.md

```diff
@@ -5,17 +5,26 @@ Notable changes to this project are documented in this file. The format is based
 ## [Unreleased]
 
 Breaking changes:
+- Added support for PureScript 0.14 and dropped support for all previous versions (#133)
+- Replaced `ComonadEnv` constraint with `ComonadAsk` in the `asks` function (#131)
 
 New features:
+- Added `IdentityT` (#121)
 
 Bugfixes:
+- Added superclasses to `MonadWriter` and `MonadTell` to enforce the type class laws (#126)
```

Comment:
I think this could be a bit clearer as to which super classes were added. Something like this?
> Added `Semigroup` and `Monoid` superclasses to `MonadTell` and `MonadWriter`, respectively, to enforce the type class laws.